10x/count/count.wdl

```diff
@@ -137,6 +137,11 @@ task chunk_reads_main {
     
     # Read the out_chunks definitions into a file
     jq '.out_chunks' _outs > out_chunks.json
+    jq -r  'map(.read_chunks | to_entries | map(.value)) | map(join(" ")) | .[]' out_chunks.json > chunk_filenames.lines
```

Comment:
Sorry I still need a bit of hand-holding for code blocks like this.

Reply:
No problem, added some explanation. I'm basically just tarring the fastq chunks, but I have to parse the JSON file to know which files to include in which tarball.

Reply:
I thought your explanation was really helpful! Thanks for adding it in @mckinsel